src/helpers.coffee

```diff
@@ -31,13 +32,16 @@ jade = async ({source, target, data}) ->
   render = _jade.compile source.content, filename: source.path
   target.content = render data
 
+handlebars = async ({source, target, data}) ->
+  source.content ?= yield read source.path
+  target.content = _handlebars source.content, data
```

Comment:
The sass and stylus helpers use promises (https://github.com/diminish7/panda-9000/blob/f649b84bdc9263d097a2f1d978a60cae4324b8cb/src/helpers.coffee#L39).

The jade helper is the only one that isn't wrapped in a promise.